### PR TITLE
1636950: Make value possibly plural

### PR DESCRIPTION
**Pull request by tlhmerry9**

### CODE REVIEW

**Comment by tlhmerry9** [comment]
To Test:

man syspurpose/man/syspurpose.8

**Review by jirihnidek — COMMENTED** [review]
The change in this PR looks OK, but this PR is blocked by PR #2490. I would wait until #2490 is merged to 1.28 branch and then I would start to deal with this PR again.

**Review by jirihnidek — APPROVED** [review]

**Pull request merged**
